### PR TITLE
{chem}[gfbf/2023a] xtb v6.7.0

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/x/xtb/xtb-6.7.0-gfbf-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    ('pkgconf', '1.9.5'),
+]
+
+configopts = "-Dlapack='custom' "
+configopts += "-Dcustom_libraries='flexiblas' "
+configopts += "--buildtype release "
+
+runtest = 'meson'
+# Ensure test don't timeout and only use one process
+testopts = 'test -C %(builddir)s/easybuild_obj -t 100  --num-processes=1'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/x/xtb/xtb-6.7.0-gfbf-2023a.eb:28]: I think the main issue is that on nodes with a lot of cores the tests use too many threads, in that case

```python
runtest = 'meson'
pretestopts = 'export OMP_NUM_THREADS=2 && '
testopts = 'test -C %(builddir)s/easybuild_obj -t 60'  # Ensure test don't timeout
```

for example, may be a better option (?)

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node301.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/1738a2b2de6b6ecc4a91fc49cd306b15 for a full test report.

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen3

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @WilleBell!

**Pull request merged**